lib/modules/bulk_data/bulk_data_export_sequence.rb

```diff
@@ -154,10 +159,10 @@ def check_cancel_request
         end
 
         @client.set_no_auth
-        skip 'Could not verify this functionality when bearer token is not set' if @instance.token.blank?
+        skip 'Could not verify this functionality when bearer token is not set' if @instance.bulk_access_token.blank?
```

Comment:
Don't most of the bulk data tests need a skip like this?

Reply:
bulk data allows open access without access token